packages/features/bookings/Booker/Booker.tsx

```diff
@@ -124,24 +126,21 @@ const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: Book
       )}
       <div className="flex h-full w-full flex-col items-center">
         <div
-          style={resizeAnimationConfig[layout]?.[bookerState] || resizeAnimationConfig[layout].default}
+          ref={animationScope}
           className={classNames(
             // Size settings are abstracted on their own lines purely for readbility.
-            // General sizes:
             "[--booker-main-width:480px] [--booker-timeslots-width:240px] lg:[--booker-timeslots-width:280px]",
-            "bg-muted grid max-w-full auto-rows-max items-start overflow-clip dark:[color-scheme:dark] md:flex-row",
```

Comment:
Was able to remove quite a few of these default classes by moving the animation logic to the new hook down below <3